commerce-account-admin-web/src/main/java/com/liferay/commerce/account/admin/web/internal/portlet/action/EditCommerceAccountUserRelMVCActionCommand.java

```diff
@@ -135,10 +139,29 @@ else if (cmd.equals(Constants.DELETE)) {
 		}
 	}
 
+	protected void updateCommerceUserRoles(ActionRequest actionRequest)
+		throws Exception {
+
+		long commerceAccountGroupId = ParamUtil.getLong(
+			actionRequest, "commerceAccountGroupId");
+
+		long commerceAccountUserId = ParamUtil.getLong(
+			actionRequest, "commerceAccountUserId");
+
+		long[] roleIds = StringUtil.split(
+			ParamUtil.getString(actionRequest, "roleIds"), 0L);
```

Comment:
should be able to do `long[] roleIds = ParamUtil.getLongValues....` rather than getting a string and splitting it